pkgs/applications/networking/cluster/helm/default.nix

```diff
@@ -2,16 +2,16 @@
 
 buildGoModule rec {
   pname = "helm";
-  version = "3.8.1";
+  version = "3.8.2";
   gitCommit = "5cb9af4b1b271d11d7a97a71df3ac337dd94ad37";
```

Comment:
Let's change this one as well.

Reply:
Thanks for catching that :) I've just updated.